contrib/tiramisu/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "tiramisu"
+pkgver = "2.0.20240610"
+pkgrel = 0
+build_style = "makefile"
+hostmakedepends = [
+    "pkgconf",
+]
+makedepends = [
+    "vala",
+    "glib-devel",
+]
+depends = [
+    "glib",
+    "zlib-ng-compat",
+    "libffi8",
+    "libblkid",
+    "libpcre2",
+    "libmount",
+]
```

Comment:
```suggestion
```

the ones that are used (glib) are already traced:  


```
0:00:33.957 =>   +so:libc.so
0:00:33.957 =>   +so:libgio-2.0.so.0
0:00:33.957 =>   +so:libglib-2.0.so.0
0:00:33.957 =>   +so:libgobject-2.0.so.0
```

the rest of them are not used at all

Reply:
I am so surprised how magical cports is, this is so nice !